jdi-light-bootstrap-tests/src/main/java/io/github/com/pages/BootstrapFormPage.java

```diff
@@ -0,0 +1,12 @@
+package io.github.com.pages;
+
+import com.epam.jdi.light.elements.composite.WebPage;
+import com.epam.jdi.light.elements.pageobjects.annotations.locators.UI;
+import io.github.com.sections.form.FormValidationSection;
+
+public class BootstrapFormPage extends WebPage {
+
+    @UI("body")
```

Comment:
In this case locator is unnecessary. 'Cause in FormValidationSection all elements have IDs

Reply:
Done